### ISSUE TITLE
Wrong highlighting for italic and headings

### ISSUE THREAD

**Issue by gruenerBogen** [serlo/mwlint#43]
The following notation is highlighted incorrectly:
 - `#` is highlighted as a heading although headings are enclosed in `==`
 - Text enclosed in `*` or `_` is highlighted as italic although italic text is enclosed in `''`

**Comment by vroland** [comment]
Yes, that is because we use the standard markdown highlighting for the code view. This could be fixed by writing our own highlighting rules. I'm leaving it open as 'enhancement' for now, as it does not impact core functionality.
